### PR TITLE
GTTP2: New embedded template

### PR DESCRIPTION
**Pull request by tadler**
# What

A new template and design for the embedded version of the Voting Information Tool! 
It's neater and cleaner and, for now, totally uncustomizable! 

# Why

As it previously existed, the old embeddable tool was broken and busy, with more customization options than we cared to maintain. 

# In the future

We'll add back some simpler customization options -- possibly a small number of background image choices, each with a corresponding accent color.

gttp-embeddable-sketch

Pivotal: https://www.pivotaltracker.com/story/show/160074179

Thanks as ever to @ddellacosta and @tank157 for their help!

cc @MariaBianchi

### CODE REVIEW

**Comment by tadler** [comment]
@ddellacosta @tank157 I'd really like to change the input placeholder from "Enter a location" to "Enter your address" for clarity, but I wasn't able to track that down.

I also wouldn't mind changing "Search" to "Find your polling location" on wider screens, but that's really just a nice-to-have and shouldn't hold up this PR.

File: src/cljs/gttp2/views/toolbar.cljs [hunk 1]
```diff
@@ -32,27 +32,34 @@
...
                            "display-none"
                            (if landing-screen?
                              "language-options-dark"
                              "language-options-light"))]
-    [:div.toolbar
-      [:div.toolbar-bttn.toolbar-lang-menu{:class themed-nav-classes}
-      [:div.language-options-selected {:on-click #(swap! close-menu? not)}
-        [:i.material-icons.language-menu-icon {:class (if landing-screen? "" "language-menu-icon-light")} "language"]
-        [:div
+    [:div.toolbar {:class (if embedded? "embedded--toolbar" "")}
```
Comment (tank157) [inline src/cljs/gttp2/views/toolbar.cljs]: In situations like this where you either are specifying classes or none, it's probably better to write it like:

`[:div.toolbar (when embedded? {:class "embedded--toolbar"}) ...`

It should do the right thing in the non-embedded case too.

File: src/cljs/gttp2/views/toolbar.cljs [hunk 2]
```diff
@@ -32,27 +32,34 @@
...
+     [:div.toolbar-bttn.toolbar-lang-menu
+      {:class (if embedded? "embedded--toolbar_button" themed-nav-classes) }
+      [:div.language-options-selected 
+        {:class (if embedded? "embedded--language-options_selected" "")
+         :on-click #(swap! close-menu? not)}
+       [:i.material-icons.language-menu-icon 
+        {:class (if landing-screen? "" "language-menu-icon-light")} "language"]
```
Comment (tank157) [inline src/cljs/gttp2/views/toolbar.cljs]: Here you could use `(when-not landing-screen? {:class "language-menu-icon-light"})` for instance

File: src/cljs/gttp2/views.cljs [hunk 3]
```diff
@@ -90,15 +92,47 @@
...
+  [lang title]
+  (let [close-menu? (r/atom true)
+        logo-url @(re-frame/subscribe [::subs/logo-url])]
+    
+    [:div.embedded--landing-container
+
+      [:header.embedded--landing-header
+        [:img.embedded--logo-img {:src logo-url}]
+        [:div
+          [:span.embedded--small-heading "The Voting Information Project"]
```
Comment (tank157) [inline src/cljs/gttp2/views.cljs]: We should use the translate mechanism to move these strings out so we can translate them

Comment (tank157) [reply src/cljs/gttp2/views.cljs]: Looks like some of them were maybe already added to the dictionary?

Comment (ddellacosta) [reply src/cljs/gttp2/views.cljs]: @tank157 Yeah I think these may be the ones that we added together when we paired on this. @tadler is there something preventing us from using those now? I agree with Troy that we should use the translations if at all possible.

**Comment by ddellacosta** [comment]
@tadler I wonder if the placeholder thing is a browser default? I don't see that we're setting a placeholder anywhere, and additionally I see it getting translated automatically across the board for me when I switch languages. Maybe that just needs a specific `:placeholder` attribute added to that `input` element (keep in mind it also needs to be added as a translation too if you add it in).

File: src/cljs/gttp2/views/toolbar.cljs (hunk 1 above)
Comment (tadler) [reply src/cljs/gttp2/views/toolbar.cljs]: `when`! Cool. I'll update!

File: src/cljs/gttp2/views.cljs (hunk 3 above)
Comment (tadler) [reply src/cljs/gttp2/views.cljs]: @ddellacosta Yeah they seemed to break after you and I paired for some reason? I'll try to work this out again, stay tuned.

**Pull request merged**